src/constants.js

```diff
@@ -1,5 +1,5 @@
 export const DOMAIN = 'accessible-colors.com';
 export const SITE_URL = `http://${DOMAIN}`;
 export const REPO = 'moroshko/accessible-colors';
-export const MIN_FONT_SIZE = 10;
-export const MAX_FONT_SIZE = 21;
+export const MIN_FONT_SIZE_IN_PX = 8;
+export const MAX_FONT_SIZE_IN_PX = 36;
```

Comment:
Good question. Why do we have constraints at all? As long as the preview handles large fonts  nicely (num lines to show), we shouldn't have a constraint at either end.

Remove the constraints?